src/parsers/junit.js

```diff
@@ -44,18 +44,23 @@ function setAggregateResults(result) {
     let failed = 0;
     let errors = 0;
     let skipped = 0;
+    let duration = 0;
     result.suites.forEach(_suite => {
       total = _suite.total + total;
       passed = _suite.passed + passed;
       failed = _suite.failed + failed;
       errors = _suite.errors + errors;
       skipped = _suite.skipped + skipped;
+      duration = _suite.duration + duration;
```

Comment:
Total duration is not always the sum of suite durations. For example, if two suites run in parallel then the total duration is the max of them. I see you are initialising it when the `result.duration` is NaN and hoping it will not cause any issue.